### PR TITLE
feature: Release notes for Codacy Cloud February 2022 DOCS-326

### PR DESCRIPTION
**Pull request by prcr**
Adds auto-generated release notes for Codacy Cloud February 2022

### :eyes: Live preview
https://github.com/codacy/docs/blob/feature/release-notes-cloud-2022-02/docs/release-notes/cloud/cloud-2022-02.md
(Unfortunately, there's no pretty live preview because we ran out of free build minutes on Netlify. :cry:)

### 🚧 To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Update release date and remove `TODO` comments

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2022-02.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+These release notes are for the Codacy Cloud updates during February 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   It's now possible to click on the organization overview chart to drill down on the information and understand which repositories require attention to improve the organization's code quality. (CY-5580)
+
+    ![Drilling down on the Overall quality chart](../images/cy-5580.png)
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2022-02.md:20]: @RaquelHipolito Should we mention something about this being a beta feature? We already have a note on the [documentation page for the Organization Overview](https://docs.codacy.com/organizations/organization-overview/#overall-quality-chart).

Comment (terciosoares) [reply docs/release-notes/cloud/cloud-2022-02.md:20]: As it is already documented I don't find it to be very detailed, but you could mention it here, maybe just by a tag.

**Review by RaquelHipolito — COMMENTED** [review]
Looks good to me @prcr. Thanks 😊

File: docs/release-notes/cloud/cloud-2022-02.md (hunk 1 above)
Comment (prcr) [reply docs/release-notes/cloud/cloud-2022-02.md]: I added `(Beta)` at the end of the release note text.

**Pull request merged**